### PR TITLE
[W5.3b][W13-A1]Raynold Ng

### PR DESCRIPTION
**Pull request by raynoldng**
LO-ClassLevel

### CODE REVIEW

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -10,6 +10,12 @@
  */
 public class Person implements ReadOnlyPerson {
 
+    public static int nextSequenceNumber = 0;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Person.java:13]: It would make semantic sense to start with 1 rather than 0!

**Review by okkhoy — COMMENTED** [review]
Incorrect PR title :stuck_out_tongue:

**Comment by okkhoy** [comment]
@raynoldng 
Some comments added. Please ack & close the PR after reading comments.

**Comment by raynoldng** [comment]
Acknowledged with thanks

**Pull request closed without merging**